tools/unicodeCat.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const request = require('request');
+
+create(
+    'bmpLetters',
+    'https://raw.githubusercontent.com/mathiasbynens/unicode-6.2.0/' +
```

Comment:
We could make the `unicode-6.2.0` package a dependency of ours, but that package is pretty big, and we only need a tiny portion of that, plus we only need that tiny portion if we decide to twiddle with the regexp generated from it. We could of course download the file and add it to our codebase, but then we'd have to include licensing and attribution details as well. And the file is still over 600kB. The most likely reason one would want to run my script is in order to switch to e.g. Unicode 8, in which case changing a URL is probably easier than replacing a file in our repo, particularly with respect to the resulting diff.

In general I'd say the repository is for things unique to our project, everything external should be fetched externally. That's why we removed the Closure Compiler from our repository when we refactored it before going public. That's why we don't include GWT or Node or any of the things we need to compile.

I just read [the unicode-data readme](https://github.com/mathiasbynens/unicode-data#readme), from where this data presumably originated. It mentions a [HTTP API](https://github.com/mathiasbynens/unicode-data#http-api) which we might use instead of this GitHub download. I could also follow https://github.com/mathiasbynens/node-unicode-data/issues/14 and create a node package just for the `L` category codepoint list. I don't feel like maintaining all categories for all Unicode versions, though. Or we could depend on `node-unicode-data` and generate the data file ourselves.

Since we should always be able to get the list of codepoints for a given category from some other source if one source fails us, I see no urgent need to add that file to the repo. Have I convinced you? Do you prefer any of the other alternatives I mentioned?